packages/experiments/src/components/Chiclet/ChicletXsmall.styles.tsx

```diff
@@ -0,0 +1,62 @@
+import { normalize } from '../../Styling';
+import { IChicletXsmallStyleProps, IChicletXsmallStyles } from './ChicletXsmall.types';
+
+export const getStyles = (props: IChicletXsmallStyleProps): IChicletXsmallStyles => {
+  const { theme, className } = props;
+  const { palette } = theme;
+
+  return {
+    root: [
+      'ms-ChicletXsmall',
+      theme.fonts.xSmall,
+      normalize,
+      {
+        backgroundColor: palette.white,
+        boxShadow: '0 1px 3px 0 rgba(0, 0, 0, 0.3)',
+        padding: 4,
+        width: 250,
+        height: 103,
+        userSelect: 'none',
+        overflow: 'hidden',
+        position: 'relative',
+        selectors: {
+          ':hover': {
+            cursor: 'pointer'
+          }
+        }
+      },
+      className
+    ],
+    icon: [
+      {
+        float: 'left',
```

Comment:
float is so old school 😄 ... I wonder if a flex box model can be implemented here?